src/main/java/org/broadinstitute/hellbender/utils/pairhmm/DragstrReadSTRAnalizer.java

```diff
@@ -0,0 +1,182 @@
+package org.broadinstitute.hellbender.utils.pairhmm;
+
+import org.broadinstitute.hellbender.utils.MathUtils;
+
+import java.util.Arrays;
+
+public class DragstrReadSTRAnalizer {
```

Comment:
I think you should add a javadoc explainaing what exactly is in the tables that you pull here. A concise description somewhere that explains how the forwards and reverse looking works and what exactly "numberOfMostRepeats()" is for a given position would be very helpful i think.